mmv1/third_party/terraform/services/logging/resource_logging_bucket_config_test.go

```diff
@@ -523,3 +523,69 @@ func getLoggingBucketConfigs(context map[string]interface{}) map[string]string {
 	}
 
 }
+
+func TestAccLoggingBucketConfigProject_indexConfigs(t *testing.T) {
```

Comment:
Can you add a test on one of the other targets? It's not a big deal if you can't run it locally, since I'll be able to in our CI.